lib/slappy/listeners/concerns/listenable.rb

```diff
@@ -0,0 +1,26 @@
+module Slappy
+  module Listener
+    module Listenable
+      include ActiveSupport::Concern
+
+      attr_reader :pattern
+
+      def initialize(target, callback)
+        @pattern = target
+        @callback = callback
+      end
+
+      def call(event)
+        return unless time_valid?(event)
+        target_element = self.class.name.split('::').last.gsub(/Listener$/, '').underscore.to_sym
+        event.matches = event.send(target_element).match pattern
+        return unless event.matches
+        @callback.call(event)
+      end
+
+      def time_valid?(event)
+        Time.at(event.ts.to_f) > Slappy.client.start_time
```

Comment:
Rails/TimeZone: Do not use `Time.at` without zone. Use one of `Time.zone.at`, `Time.at.current`, `Time.at.in_time_zone`, `Time.at.utc`, `Time.at.getlocal`, `Time.at.iso8601`, `Time.at.jisx0301`, `Time.at.rfc3339`, `Time.at.to_i`, `Time.at.to_f` instead.